### ISSUE TITLE
Error on line 171 - Decision Trees

### ISSUE THREAD

**Issue by jacksusank** [rtealwitter/datamining2026spring#22]
I think this statement is misleading/false:

Old Line: "Recall that the gradient descent update subtracts the gradient of the parameter (opposite the direction of steepest ascent) for a small learning rate $\alpha$."

Suggested fix: "Recall that gradient descent updates parameters by subtracting the gradient of the loss with respect to those parameters (opposite the direction of steepest ascent) for a small learning rate $\alpha$."
